internal/factsengine/gatherers/corosyncconf_test.go

```diff
@@ -98,42 +98,59 @@ func (suite *CorosyncConfTestSuite) TestCorosyncConfBasic() {
 		{
 			Name:  "corosync_not_found",
 			Value: nil,
-			Error: nil,
+			Error: &entities.FactGatheringError{
+				Message: "requested field value not found: totem.not_found",
+				Type:    "corosync-conf-value-not-found",
+			},
 		},
 	}
 
 	suite.NoError(err)
-	suite.ElementsMatch(expectedResults, factResults)
+	suite.ElementsMatch(expectedResults, factsGathered)
 }
 
 func (suite *CorosyncConfTestSuite) TestCorosyncConfFileNotExists() {
 	c := NewCorosyncConfGatherer("not_found")
 
-	factRequests := []entities.FactRequest{
+	factsRequest := []entities.FactRequest{
 		{
 			Name:     "corosync_token",
 			Gatherer: "corosync.conf",
 			Argument: "totem.token",
 		},
 	}
 
-	_, err := c.Gather(factRequests)
+	factsGathered, err := c.Gather(factsRequest)
+
+	expectedError := &entities.FactGatheringError{
```

Comment:
Not sure about this one, but shouldn't it be `FactsGatheringError` (in plural)?

Reply:
No, it is one fact gatherer error.